sdks/java/io/jdbc/src/main/java/org/apache/beam/sdk/io/jdbc/JdbcIO.java

```diff
@@ -557,6 +562,18 @@ public DataSourceConfiguration withConnectionInitSqls(
       return builder().setConnectionInitSqls(connectionInitSqls).build();
     }
 
+    /** Sets the maximum total number of connections. */
```

Comment:
Please, add a comment that negative value is used for no limit.

Reply:
Thanks @aromanenko-dev  for the review, Added specified changes.